textproc/wc2/Portfile

```diff
@@ -34,6 +34,11 @@ checksums           rmd160  8bf38fa4b3f2a96f4e1c5191cbbfe2024463d523 \
                     sha256  f7d8cc810c4a4a61ada54996b523fe4877f040d3b33dc77df57cb3687d4ae3b0 \
                     size    31172
 
+# cc1: error: unrecognized command line option "-Wpedantic"
+post-patch {
+    reinplace "s|-Wpedantic|-pedantic|g" ${worksrcpath}/Makefile
```

Comment:
Please use a patchfile, not `reinplace`.

Reply:
Done